distribution/docker/distroless-libc/Dockerfile

```diff
@@ -1,7 +1,9 @@
 FROM debian:bullseye-slim AS builder
 
+WORKDIR /vector
+
 COPY vector_*.deb ./
-RUN dpkg -i vector_*_$(dpkg --print-architecture).deb
+RUN dpkg -i "vector_*_$(dpkg --print-architecture).deb"
```

Comment:
And these! :trollface:

Reply:
I'll sweep through for all the `$(...)`s and make sure they're quoted properly, interesting that the linter doesn't yell about these though.

Reply:
> interesting that the linter doesn't yell about these though.

The linter has no way to know what is the actual intention here (that's why it's a false positive).
Quoting the whole thing is a valid intention, as it is not quoting it.

Reply:
Fixed c78fa130d